tools/packager/packager.py

```diff
@@ -73,6 +76,14 @@ def run_packager():
     if args.cmd == "help" or not args.cmd:
         print(USAGE)
 
+    elif args.cmd == "version":
+        try:
+            cmd = shlex.split("git log -1 --oneline")
+            p = Popen(cmd, stdout=PIPE).communicate()[0]
```

Comment:
`subprocess.call` does the same job.

Reply:
Yes.  subprocess.call is a wrapper around Popen.  Are you requiring this change, or is what I have good enough?

Reply:
what about something like this:
```Python
subprocess.check_output('git describe --tags', shell=True).split()[0]
``` 
which yields something like:
```
'0.9.0-296-g777469f'
```
Note that `check_output` is a wrapper around Popen too but it's easier if you just want to get the stdout. Also, note that passing in the kwarg `shell=True` removes the requirement of using `shlex`.

Reply:
@asgibbons `call` automatically handles kill and wait which gracefully get rid of dead processes. But like @r-rojo pointed out, I've made mistake, it should be `check_output`.